miniwallet/config.ts

```diff
@@ -0,0 +1,146 @@
+import { ethers } from 'ethers'
+import 'dotenv/config'
+
+export default {
+  mainnet: {
+    miniWallet: {
+      initialOperatorThreshold: process.env.MINIWALLET_INITIAL_OPERATOR_THRESHOLD,
+      initialOperators: JSON.parse(process.env.MINIWALLET_INITIAL_OPERATORS || '[]'),
+      initialUserLimit: ethers.utils.parseEther(process.env.MINIWALLET_INIITIAL_USER_LIMIT || '1000000'),
+      initialAuthLimit: ethers.utils.parseEther(process.env.MINIWALLET_INIITIAL_AUTH_LIMIT || '100000')
+    },
+    miniID: {
+      baseUri: process.env.MINIID_BASE_URI || 'ipfs://Qmc8DVEthq7cZMTMyZ2NQ8dHkG99n549DMBwNzAypQgXe1/MiniID'
+    },
+    mini721: {
```

Comment:
721, 1155, and miniId are extensions to core mini-wallet features - they can be optional. Let's move them to separate config files

Reply:
Refactored approach and commit are [here](https://github.com/polymorpher/sms-wallet/issues/13#issuecomment-1246191757)